Program.cs

```diff
@@ -538,11 +538,10 @@ static void Trade(string user = "", string passkey = "")
 
 			foreach (var account in SteamGuardAccounts)
 			{
-				if (user != "")
-					if (!string.Equals(account.AccountName, user, StringComparison.CurrentCultureIgnoreCase))
-						break;
```

Comment:
I think just changing this line to a `continue` would suffice.

Reply:
That would work too, it's up to you. I took a look at how `accept-all` was implemented and just did something similar because I personally don't like using `continue` unless necessary.